src/services/config.js

```diff
@@ -0,0 +1,285 @@
+   *
+   * getEnv('example', 'my_setting') === process.env.EXAMPLE_MY_SETTING
+   */
+  static getEnv (prefix, name) {
+    let envVar
+    if (name && prefix) envVar = `${prefix}_${name}`
+    else if (name && !prefix) envVar = name
+    else if (!name && prefix) envVar = prefix
+    else throw new TypeError('Invalid environment variable')
+
+    return process.env[envVar.toUpperCase().replace(/-/g, '_')]
+  }
+
+  generateDefaultPairs (accounts) {
+    return Utils.getPairs(accounts).map((pair) => {
+      return pair.map((desc) => desc.ledger)
+    })
+  }
+
+  parseCredentials () {
+    const credentialsEnv = this.parseCredentialsEnv()
+
+    return _.mapValues(credentialsEnv, (credentials) => {
+      // DEPRECATED: `account_uri` (should be just `account`)
+      if (credentials.account_uri) {
+        console.error('DEPRECATED: The key `account_uri` in ledger credentials has been renamed `account`')
+        credentials.account = credentials.account_uri
+        delete credentials.account_uri
+      }
+
+      const isClientCertCredential = credentials.key !== undefined
+
+      if (isClientCertCredential) {
+        return _.omitBy(_.assign(credentials, {
+          key: fs.readFileSync(credentials.key),
+          cert: fs.readFileSync(credentials.cert),
+          ca: credentials.ca && fs.readFileSync(credentials.ca)
+        }), _.isUndefined)
+      }
+
+      return _.omitBy(_.assign(credentials, {
+        ca: credentials.ca && fs.readFileSync(credentials.ca)
+      }), _.isUndefined)
+    })
+  }
+
+  parseCredentialsEnv () {
+    if (Config.getEnv(envPrefix, 'LEDGERS')) {
+      log.error('You set CONNECTOR_LEDGERS, which is a deprecated configuration variable. Please migrate to CONNECTOR_ACCOUNTS.')
+      process.exit(1)
+    }
+
+    return JSON.parse(Config.getEnv(envPrefix, 'ACCOUNTS') || '{}')
+  }
+
+  parseRoutes () {
+    const routes = JSON.parse(Config.getEnv(envPrefix, 'ROUTES') || '[]')
+
+    for (let route of routes) {
+      if (
+        typeof route.peerAddress !== 'string' ||
+        !route.peerAddress.match(/^[a-zA-Z0-9._~-]+$/) ||
+        typeof route.targetPrefix !== 'string' ||
+        !route.peerAddress.match(/^[a-zA-Z0-9._~-]*$/)
+      ) {
+        throw new Error('invalid format for CONNECTOR_ROUTES. value=' +
+        JSON.stringify(route))
+      }
+    }
+
+    return routes
+  }
+
+  getLocalConfig () {
+    // Routes to add to the connector, in the form:
+    // [{
+    //  "targetPrefix": "", // match any route
+    //  "peerAddress": "ilpdemo.red."
+    // }, {
+    //  "targetPrefix": "usd.",
+    //  "peerAddress": "example.other."
+    // }]
+    this.routes = this.parseRoutes()
+
+    const features = this.features = {}
+    // Debug feature: Reply to websocket notifications
+    features.debugReplyNotifications =
+    Config.castBool(Config.getEnv(envPrefix, 'DEBUG_REPLY_NOTIFICATIONS'))
+
+    this.address = Config.getEnv(envPrefix, 'ILP_ADDRESS') || ''
```

Comment:
This parameter should be documented in the README.

Reply:
Yeah, README needs a major rewrite...